src/components/Articles.jsx

```diff
@@ -18,6 +18,7 @@ function Articles(props) {
         setLoaded(true);
         let data = result.data.articles;
         if (num) {
+          data = data.splice(0, num);
```

Comment:
I'm not too sure what this is doing Beth. This component is never passed any props - If this is to isolate an individual article, you should really be requesting ONLY that article rather than more data than necessary only to trim it down